lib/webapp.js

```diff
@@ -657,7 +691,7 @@ const staticHandlers = {
         return res.status(405).json({error: 'ZWED0129E - Only POST method supported'});
       });
       router.get('/environment', function(req, res){
-        getUserEnv().then(result => {
+        getUserEnv(rbac).then(result => {
```

Comment:
I think you're effectively checking for rbac twice and duplicating code, and will have to do it this way for each subsequent endpoint that works w/ rbac and partially works w/o rbac.

What you could do instead is, use `router.get('/environment' [...]` **once** outside the main if/else, then leave the actual conditional endpoints to the conditional check